.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+### 🗒 Description
+
+If helpful for the review process, please describe your changes from a technical or non technical perspective
+
+#
+### 🎫 Git ticket reference
+
+- 
+
+#
+### 📷 Demo screen recording or Screenshots
+
+- [x] Not applicable
+
+# 
+### 📋 Checklist
+
+You may remove tasks that do not make sense in this PR.
+
+- [ ] GitHub **labels** are assigned
```

Comment:
Its better to rearrange them in order of how often a particular check is used, e.g. you can start with checking all ACs, then move on to tested and so on.